src/Symfony/Component/ErrorRenderer/Resources/views/logs.html.php

```diff
@@ -22,7 +22,7 @@
         <tr class="status-<?= $status; ?>" data-filter-level="<?= strtolower($this->escape($log['priorityName'])); ?>"<?php if ($channelIsDefined) { ?> data-filter-channel="<?= $this->escape($log['channel']); ?>"<?php } ?>
             <td class="text-small" nowrap>
                 <span class="colored text-bold"><?= $this->escape($log['priorityName']); ?></span>
-                <span class="text-muted newline"><?= (new \DateTime($log['timestamp']))->format('H:i:s'); ?></span>
+                <span class="text-muted newline"><?= (new \DateTime())->setTimestamp($log['timestamp'])->format('H:i:s'); ?></span>
```

Comment:
for short `date('H:i:s', $log['timestamp'])`?

Reply:
Yes, it's more readable 😃